ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -387,60 +320,33 @@ export const CohortSearch = withCurrentCohortSearchContext()(class extends React
     saveCriteria([param]);
   }
 
-  get showDataBrowserLink() {
-    return [Domain.CONDITION, Domain.PROCEDURE, Domain.MEASUREMENT, Domain.DRUG]
-    .includes(this.props.cohortContext.domain);
-  }
-
   render() {
-    const {cohortContext} = this.props;
-    const {count, growlVisible, selectedIds, selections, title} = this.state;
+    const {cohortContext, cohortContext: {domain, type}} = this.props;
```

Comment:
do we need 2 cohortContext?

Reply:
So if I use the destructuring syntax (`cohortContext: {domain, type}`), I can't use the `cohortContext` part of that, so I have to declare it once to get the full object and once to get the child properties.

Reply:
i see the cohortContext now thanks